Plugins/PDCPlugin/PDCPlugin.swift

```diff
@@ -214,18 +357,46 @@ struct ModuleBuildRequest {
             environment["IPHONEOS_DEPLOYMENT_TARGET"] = nil
 
             process.environment = environment
-            process.executableURL = URL(filePath: clang.path.string)
+            process.executableURL = URL(filePath: clangPath)
             process.arguments = ["-g"] + arguments
             if verbose { process.print() }
             try process.run()
             process.waitUntilExit()
             guard process.terminationStatus == 0 else { throw Error.clangFailed(exitCode: process.terminationStatus) }
         }
+        
+        @Sendable func ar(workingDir: String? = nil, _ arguments: [String]) throws {
+            let ar = try context.tool(named: "ar")
+            let process = Process()
+            process.executableURL = URL(filePath: ar.url.path(percentEncoded: false))
+            if let workingDir {
+                process.currentDirectoryURL = URL(fileURLWithPath: workingDir)
+            }
+            process.arguments = arguments
+            if verbose { process.print() }
+            try process.run()
+            process.waitUntilExit()
+            guard process.terminationStatus == 0 else { throw Error.arFailed(exitCode: process.terminationStatus) }
+        }
+        
+        @Sendable func ranlib(workingDir: String? = nil, _ arguments: [String]) throws {
+            let ar = try context.tool(named: "ranlib")
+            let process = Process()
+            process.executableURL = URL(filePath: ar.url.path(percentEncoded: false))
+            if let workingDir {
+                process.currentDirectoryURL = URL(fileURLWithPath: workingDir)
+            }
+            process.arguments = arguments
+            if verbose { process.print() }
+            try process.run()
+            process.waitUntilExit()
+            guard process.terminationStatus == 0 else { throw Error.arFailed(exitCode: process.terminationStatus) }
+        }
 
         func pdc(_ arguments: [String]) throws {
             let process = Process()
             process.executableURL = URL(filePath: "\(playdateSDK)/bin/pdc")
-            process.arguments = ["--skip-unknown"] + arguments
```

Comment:
any reason why this was removed? Without it the debug symbols are copied into the pix

Reply:
With that flag on, users can't have their own custom file types or any files the pdc doesn't compile. For example 3D models wouldn't be copied.

I'll add an extra step to delete the debug symbols from the finished pdx.